test/fixtures/output/network/dns-forwarder/describe.golden

```diff
@@ -0,0 +1,9 @@
++----------------+------------------+
+| ID             | dnsf-abcde1      |
+| Display Name   | my-dns-forwarder |
+| Domains        | abc.com, def.com |
+| DNS Server IPs | 10.200.0.0       |
+| Environment    | env-00000        |
+| Gateway        | gw-111111        |
+| Phase          | READY            |
++----------------+------------------+
```

Comment:
@kabraham13 to confirm output format

Reply:
Ah I see your point, if there are a lot of domains this field just extends to the right in the describe command (in list the `Domains` column is capped at the length of the longest domain).

Alternatively we can print each domain on a separate line:

```
+----------------+------------------------+
| ID             | dnsf-abcde1            |
| Display Name   | my-dns-forwarder       |
| Domains        | abc.com                |
|                | def.com                |
|                | example.domain         |
|                | another.example.domain |
|                | foo.com                |
|                | bar.com                |
| DNS Server IPs | 10.200.0.0             |
| Environment    | env-00000              |
| Gateway        | gw-111111              |
| Phase          | READY                  |
+----------------+------------------------+
```

Reply:
@belringer what is your opinion here?

Reply:
It is definitely more legible with a separate line per domain. But how would the table behave if there are 5/6/10 or more domains separated by commas? Are they listed on one line and the table expands to the right indefinitely? Or is there some line break or a cut-off like "..." at some point?

Reply:
Table extends down with column size capped to the length of the longest domain. Here they are for comparison (sorted as well). 
```
+----------------+----------------------------------+
| ID             | dnsf-abcde1                      |
| Name           | my-dns-forwarder                 |
| Domains        | abc.com, def.com,                |
|                | example.com, example.domain,     |
|                | my.dns.forwarder.example.domain, |
|                | xyz.com                          |
| DNS Server IPs | 10.200.0.0                       |
| Environment    | env-00000                        |
| Gateway        | gw-111111                        |
| Phase          | READY                            |
+----------------+----------------------------------+

+----------------+---------------------------------+
| ID             | dnsf-abcde1                     |
| Name           | my-dns-forwarder                |
| Domains        | abc.com                         |
|                | def.com                         |
|                | example.com                     |
|                | example.domain                  |
|                | my.dns.forwarder.example.domain |
|                | xyz.com                         |
| DNS Server IPs | 10.200.0.0                      |
| Environment    | env-00000                       |
| Gateway        | gw-111111                       |
| Phase          | READY                           |
+----------------+---------------------------------+
```

Reply:
To be honest, I don't have a strong preference and I think both work. If its crucial that each domain can easily be recognized by users then the line separator is better. If vertical space should be saved to easily get an overview of the result then the comma separator is more favourable.

Reply:
My vote is for a comma-delimited list since that is standard across all of our other commands. Happy to make an exception though if there's a strong preference for the other way!

Reply:
Lets go with the standard for now then - no need to break the pattern for this